docs/pages/machine-id/guides/hostcertificate.mdx

```diff
@@ -0,0 +1,373 @@
+- `tctl` is the administrative tool you will use to create the bot user.
+- `tsh` is the client tool you will use to log in to the OpenSSH server.
+- `tbot` is the Machine ID tool you will use to associate a bot user with a machine.
+
+Machine ID is available starting from the Teleport 9.0.0 release. Download the appropriate Teleport package for your platform from our [downloads page](https://goteleport.com/teleport/download).
+
+Next, log in to your cluster from the remote device using `tsh` to ensure that your device will be able to interact with the cluster:
+
+```code
+tsh login --proxy=myteleportcluster.com --user=teleport-username
+```
+
+
+## Step 2/4. Create a Role to Issue `tbot` Certificates
+
+In order to create certificates `tbot` requires a role that provides the necessary permissions. Roles may restrict which **principals** can be requested using **predicates**,
+or strings which are used to reduce the scope of permissions for creating host certificates.
+By using the format of `where: 'predicate(host_cert.principals, "domain.name")'`, a Teleport user can create a role that will only be able to able to affect the designated domain name
+in a manner defined by the selected predicate. A list of supported predicates and their behavior are listed below:
+
+| **Predicate** | **Behavior** | **Example** |
+| ----------- | ----------- | ----------- |
+| all_equal | All requested principals must be equal to the set string. If the set string is `foo.example.com`, host certificates limit access only to `foo.example.com`. | `all_equal(host_cert.principals, "foo.example.com")` |
+| all_end_with |  All requested principals must end with the set string. If the set string is `.example.com`, host certificates limit access only to `*.example.com`. | `all_end_with(host_cert.principals, ".example.com")` |
+| is_subset | All requested principals must be contained in the listed set. | `is_subset(host_cert.principals, "foo.example.com", "bar.example.com")` |
+
+In the case of an example role for use within this guide, use a text editor of your choice to create a role YAML named `tbotrole.yaml` with the following configuration options, replacing `nodename.my.domain.com` with your own domain name,
+or the domain associated with your cluster:
+
+```code
+ kind: role
+metadata:
+  name: hostcert-bot
+version: v5
+spec:
+  allow:
+    rules:
+      - resources:
+          - host_cert
+        verbs:
+          - create
+        where: 'is_subset(host_cert.principals, "nodename.my.domain.com")'
+```
+
+Once complete, create your role to apply to the tbot configuration in the next step:
+
+```code
+$ tctl create tbotrole.yaml
+```
+
+
+### Step 3/4 Create Your `tbot` Configuration
+
+Before you create a bot user, you need to determine which role(s) you want to
+assign to it. You can use the `tctl` command below to examine what roles exist
+on your system.
+
+<ScopedBlock scope={["cloud"]}>
```

Comment:
I think we can remove lines 96–103, since the `tctl.mdx` partial above already shows you how to log in to use `tctl`, and the sentence before this says "You can use the `tctl command below...".